src/app/src/main.cpp

```diff
@@ -85,19 +98,36 @@ int main(int argc, char* argv[]) {
     spdlog::flush_every(std::chrono::seconds(10));
     spdlog::set_default_logger(logger);
 
+    auto gxp_file_sink = std::make_shared<spdlog::sinks::basic_file_sink_mt>(gxpLogfile, true);
+    auto gxpLogger = std::make_shared<spdlog::logger>("nautilai_gxp", gxp_file_sink);
+    gxpLogger->set_pattern(
+        fmt::format(
+            fmt::runtime("{{\"timestamp\": \"%Y-%m-%d %H:%M:%S.%e\", \"username\": \"{}\", \"computer_name\": \"{}\", \"event\": \"%v\"}}"),
+            userName,
+            computer_name
+        ),
+        spdlog::pattern_time_type::utc
+    );
+    gxpLogger->flush_on(spdlog::level::info);
+    spdlog::register_logger(gxpLogger);
+
     //create AppData directory for config file
-    std::filesystem::path configPath = (userProfile / "AppData" / "Local" / "Nautilai");
-    std::filesystem::path configFile = (configPath / "nautilai.toml");
+    std::filesystem::path configFile = (naAppDataPath / "nautilai.toml");
+
+    auto logFn = [](std::string s) {
+        spdlog::info(s);
+        spdlog::get("nautilai_gxp")->info(s);
```

Comment:
I think I saw this somewhere else, so I wonder if it merits being moved to a common place.